nuxt.config.ts

```diff
@@ -0,0 +1,75 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+  compatibilityDate: '2024-04-03',
+  devtools: { enabled: true },
```

Comment:
Will we want this to be false?

Reply:
yes good point, have changed this now

Reply:
devtools: { enabled: process.env.NODE_ENV === 'development' },

should we have it conditionally set like this?? or just false

Reply:
I see you've already made this change so will mark as resolved